Source/Core/Core/HW/WiiSaveCrypted.cpp

```diff
@@ -189,7 +189,7 @@ void CWiiSaveCrypted::ReadHDR()
 	}
 	std::string banner_file_path = m_wii_title_path + "banner.bin";
 	if (!File::Exists(banner_file_path) ||
-		AskYesNoT("%s already exists, overwrite?", banner_file_path.c_str()))
+		AskYesNoT("%s already exists. You can still backup the current save files before overwriting.\nOverwrite now?", banner_file_path.c_str()))
```

Comment:
This only feels like it tells the user that it's possible to make a backup – it doesn't recommend them to do it like the previous message. Could we have something like "You may want to backup the current save files..."?